ner_v2/detectors/numeral/number_range/README.md

```diff
@@ -0,0 +1,246 @@
+## Number Range Detector 
+
+This detector module will help to detect minimum and maximum number values from text containing range values. For example - "200 to 300" (minimum=200 and maximum=300),  "more than 2000" (mininum=2000, maximum=Not define) 
```

Comment:
Can be simplified

This detector module will ~help to~ detect minimum and maximum number values